src/main/java/com/google/api/codegen/transformer/InitCodeImportSectionTransformer.java

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer;
+
+import com.google.api.codegen.metacode.InitCodeNode;
+import com.google.api.codegen.viewmodel.ImportSectionView;
+
+/** Generates an ImportSectionView for the InitCodeTransformer. */
+public interface InitCodeImportSectionTransformer {
```

Comment:
I split them up because Python is the only language that needs special handling of the init code import section, but many languages need special handling of the file header import section. Keeping them split up means I make fewer changes.